default/stringtables/en.txt

```diff
@@ -339,11 +339,14 @@ PROJECT_BT_STOCKPILE_SHORT_DESC
 Transfers PP into the imperial stockpile
 
 PROJECT_BT_STOCKPILE_DESC
-'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. Anything enqueued below the transfer project at a supply-connected planet will not be allocated PP.
+'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. One can specify the amount and repetition of the transfer. PP spent on the transfer are added to the imperial stockpile.
 
-Only transfers PP that are available at this planet; PP allocation on planets that are not supply-connected to here will not be affected by this project.
+Only uses PP that are produced in the same supply-group. No PP from the imperial stockpile are used.
 
-Surplus PP (PP not allocated to anything on the queue) at this or any other location will be transferred to the stockpile with or without a stockpile transfer project on the queue.  This project is used to transfer PP to the stockpile even if there are things on the queue below it that could have been allocated available PP.'''
+Surplus PP (PP not allocated to anything on the queue) at this or any other location will be transferred to the stockpile with or without a stockpile transfer project on the queue.  This project is used for controlling PP allocation to the stockpile. PP used for stockpile transfer are not available for items on the build queue below it.
```

Comment:
... allocation to the stockpile when (something that explains what is meant).

Reply:
Not sure i understand what you mean. "..allocation to the stockpile when you need more control.".?? 
I intended "PP used for stockpile transfer are not available for items on the build queue below it." to cover the part that PP spent on stockpile are not available for building in the same turn.

Would something like "This project is used for prioritising stockpile transfer in the build queue." be better?

Or "This project is used to allocate PP to the stockpile when the PP should transfer should happen before another build item."

Reply:
"This project is used for controlling PP allocation to the stockpile" is too simple and obvious a statement for where it is located in the text. It could be fine as the first sentence in the whole explanation, but not here in the midst of other text that is explaining the details.

If kept here, it needs to provide some additional information about what is meant, like that it's useful for controlling transfer to the stockpile when there is other stuff on the queue below it that could be allocated PP, or that it's useful to adjust the priority of stockpile allocation when more control is needed, without having to manipulate the stuff later on the queue (eg. by pausing or removing from the queue).

Reply:
ok, i'll just remove it